web/rootfs/defaults/settings-config.js

```diff
@@ -0,0 +1,259 @@
+{{ $XMPP_DOMAIN := .Env.XMPP_DOMAIN | default "jitsi-meet.example.com" -}}
```

Comment:
We need to sort these options somehow (alphabetically?) so it doesn't become a mess, WDYT?

Reply:
also, I'm not sure about having the defaults here. It means that the generated config will always be set and we have to keep it in sync with the internals. I'd rather pass it blindly I think, except for specific cases.

Reply:
Removed defaults.  Also sorting the options alphabetically for the time being, until a better scheme comes along.